### ISSUE TITLE
"Cutoff Answers" meaning unclear

### ISSUE THREAD

**Issue by pwim** [tokyodev-inc/tokyodev-devographics#345]
What "Cutoff Answers" means is unclear, and the text doesn't help much:

> Answers that scored below the defined threshold

Some ideas:

* Always show "Other Answers". The meaning of this is more intuitive, and ultimately there isn't really a difference from a readers perspective between "Unmatched Answers" and "Cutoff Answers"
* Rename "Cutoff Answers" to "Truncated Answers". I think "truncated" more effectively conveys what it is doing.
* Just show the rest of the answers when expanding "Other answers". In this case I'm not sure there's a good reason to be hiding them, apart from making the initial chart not too long.

**Comment by SachaG** [comment]
I could make the text more explicit, like

>    Answers that got less than 10 responses

The issue with showing everything when expanding the chart is that there might be a lot of extra answers – for example for the Country chart you could easily imagine 50+ countries, each of them with 1-2 respondents.

**Comment by pwim** [comment]
I don't think there's an inherent reason why we need to avoid showing them. But getting the UX right would be tricky. Like you'd probably want to be able to scroll through the additional answers rather than just adding them to the page.

Updating the text to be more explicit, along with renaming "Cutoff Answers" to "Truncated Answers" would probably be enough for now.

**Comment by SachaG** [comment]
<img width="363" alt="Screenshot 2024-11-11 at 11 23 18" src="https://github.com/user-attachments/assets/a97becfc-23f5-429c-aed9-1ec9bb16f63c">

I'm not sure if "truncated" really is better than "cutoff"?

**Comment by SachaG** [comment]
I think "truncate" would be a better match for the `limit` parameter (`limit = 10` means "keep the first 10 items). Whereas `cutoff` is different, `cutoff = 10` means "keep only items with more than 10 answers".

**Comment by SachaG** [comment]
If the more explicit label works for you I suggest we leave it like that for now. If we wanted to show all cutoff (or over limit) answers, even if we figured out a good UX for that (maybe pagination) we would still need to load all that extra data on the client – which for the State of XYZ surveys could be a problem since they get a lot more answers. 

I suppose we could also load the extra data on-demand dynamically, but the amount of work required doesn't really seem worth it to me right now…

**Comment by pwim** [comment]
Yeah, I can't come up with a great term. You're right that truncated answers probably isn't the right thing either.

Is there a need to show the breakdown like this, and not just "Other" instead?

**Comment by SachaG** [comment]
Not really, the only reason is if people want to click on Unmatched Answers' little comment icon to get the detail of answers that weren't matched.

**Comment by pwim** [comment]
I wonder if it would make sense to just have "Other" then, and have that be expandable only if there are unmatched answers.
